Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "foxbox"
 version = "0.1.0"
 authors = ["fabrice <fabrice@desre.org>"]
 build = "build.rs"
+exclude = [ "config/**" ]
```

Comment:
We already have a profile folder outside of the src tree where this configuration could fit better and we also have a way to specify the location of this profile folder through a [cmd line arg](https://github.com/fxbox/foxbox/blob/master/src/main.rs#L118). It would be nice if the adapter could take this location and use it to store its configuration.